scala/lambdas/ingest-asset-opex-creator/src/main/scala/uk/gov/nationalarchives/ingestassetopexcreator/Lambda.scala

```diff
@@ -99,7 +104,9 @@ class Lambda extends LambdaRunner[Input, Unit, Config, Dependencies] {
 
 object Lambda {
 
-  case class Input(id: UUID, batchId: String, executionName: String)
+  case class InputItem(id: UUID, batchId: String, assetExists: Boolean)
+
+  case class Input(items: List[InputItem])
```

Comment:
Maybe `assets` instead of `items`?

Reply:
We can't, that's what AWS provide and it's not configurable.